src/php7/ig_win32.h

```diff
@@ -1,28 +1,29 @@
-#ifndef _IG_WIN32_H
-#define _IG_WIN32_H
-
-#if PHP_WIN32
-# include "win32/php_stdint.h"
-# ifndef inline
-# define inline __inline
-# endif
-
-# if _MSC_VER >= 1900
-#  include <stdbool.h>
-# else
-#  ifndef __cplusplus
-#   if !0
-     typedef enum { false = 0, true = 1 } _Bool;
-#   endif
-#  else
-     typedef bool _Bool;
-#  endif
-#  define bool _Bool
-
-#  define false 0
-#  define true 1
-#  define __bool_true_false_are_defined 1
-# endif
-#endif
-
-#endif
+#ifndef _IG_WIN32_H
+#define _IG_WIN32_H
+
+#if PHP_WIN32
+# include "win32/php_stdint.h"
+# if defined(_MSC_VER) && _MSC_VER >= 1800
+#  include <stdbool.h>
+# else
+#  ifndef inline
+#  define inline __inline
+#  endif
+
+#  ifndef __cplusplus
+#   if !0
+t ypedef enum { false = 0, true = 1 } _BoolType;
+#  define bool _Bool
+#   endif
+#  else
+t ypedef bool _Bool;

```

Comment:
On Aug 1, 2016 7:40 AM, "Oleg Grenrus" <notifications@github.com> wrote:
>
> In src/php7/ig_win32.h:
>
> > +#if PHP_WIN32 > +# include "win32/php_stdint.h" > +# if
defined(_MSC_VER) && _MSC_VER >= 1800 > +# include <stdbool.h> > +# else >
+# ifndef inline > +# define inline __inline > +# endif > + > +# ifndef
__cplusplus > +# if !0 > +t ypedef enum { false = 0, true = 1 } _BoolType;
> +# define bool _Bool > +# endif > +# else > +t ypedef bool _Bool;
>
> typedef?

Also this part should rely on win32/php_stdint.h which should have all
types you have on gcc for c99.

If 5.3 is still supported you can bundled it and use it for 5.3 but be sure
to always use php ones for newer php versions.

> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.